test-engineering/contract-tests/kinto-setup/main.py

```diff
@@ -0,0 +1,110 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from pathlib import Path
+from time import sleep
+
+import typer
+from kinto import (
+    KintoAttachment,
+    KintoRecord,
+    create_bucket,
+    create_collection,
+    upload_attachments,
+    upload_icons,
+)
+from requests import HTTPError, Session
+
+
+def main(
+    kinto_url: str = typer.Argument(..., envvar="KINTO_URL"),
+    kinto_bucket: str = typer.Argument(..., envvar="KINTO_BUCKET"),
+    kinto_collection: str = typer.Argument(..., envvar="KINTO_COLLECTION"),
+    kinto_data_dir: Path = typer.Argument(..., envvar="KINTO_DATA_DIR"),
+):
+    """Run the CLI application."""
+
+    # Load Kinto data from the given Kinto data directory
+    kinto_records = [
+        KintoRecord(
+            record_id=data_file.stem,
+            attachment=KintoAttachment(
+                filename=data_file.name,
+                mimetype="application/json",
+                filecontent=data_file.read_bytes(),
+            ),
+        )
+        for data_file in kinto_data_dir.glob("*.json")
```

Comment:
Why do you want to have the data split into multiple files, instead of one big `data.json`?

Reply:
Because we currently have multiple records with attachments in production and I'd like the contract tests to have coverage for loading multiple records in Merino.

Reply:
makes sense, thanks!

Reply:
No problem! 👍🏻